src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -6087,17 +6082,19 @@ s32 Player_ActionHandler_13(Player* this, PlayState* play) {
                 }
 
                 sp2C = Player_ActionToBottle(this, this->itemAction);
-                if (sp2C >= 0) {
-                    if (sp2C == 0xC) {
-                        Player_SetupActionPreserveItemAction(play, this, Player_Action_8084EED8, 0);
+#define ACTION_TO_BOTTLE_ACTION(action) (action - PLAYER_IA_BOTTLE)
```

Comment:
I'd really prefer if this define was at top-level instead of in the middle of a function. Maybe in z64player.h? Then it could be used in Player_ActionToBottle as well.

(PS: I'm not really sure what to do about these "sub-enums" like with Player_ActionToMagicSpell and Player_ActionToMeleeWeapon too. Maybe they deserve a separate enum? I think we can figure that out later though)

(PPS: maybe it should be called `ACTION_TO_BOTTLE` to match the function name?)

Reply:
Yeah I would prefer to handle all these in a consistent manner at some point in the future. But if we want to introduce this now I guess we can change other examples of it later.

But there are *so* many of these id offset situations in player.

Reply:
> But there are so many of these id offset situations in player.

yeah, I kinda think this is a bigger project and not too relevant to this PR (where the goal is just to name some action funcs). Could also revert `ACTION_TO_BOTTLE_ACTION` here to really punt on it

Reply:
For now, I moved it to `z64player.h` and used it in `Player_ActionToBottle`.
I think that either this or a `PlayerBottle` enum of some kind is needed since without it it's harder to understand how it chooses an action (`sp2C >= ACTION_TO_BOTTLE(PLAYER_IA_BOTTLE_FISH) && sp2C <= ACTION_TO_BOTTLE(PLAYER_IA_BOTTLE_BUG)` vs `sp2C >= 1 && sp2C <= 3`).

Reply:
We're not saying that it isn't helpful, but rather that this pattern is a much larger thing that needs to be handled within player. And that it makes more sense to do it separately as a larger project. 
I would prefer if it was taken out temporarily, with the understanding that this enum-range pattern will be handled in the future at a larger scale.